opus/src/components/formPostOffer/FormPostOffer.js

```diff
@@ -174,4 +188,4 @@ class FormPostOffer extends React.Component {
   }
 }
 
-export default FormPostOffer;
+export default connect(mapDispatchToProps)(FormPostOffer);
```

Comment:
Reminder of the previous comment : first parameter to 'connect' function brings state, second one (if provided) brings dispatch